package.json

```diff
@@ -90,7 +90,7 @@
     "flexboxgrid": "^6.3.0",
     "immutable": "^3.8.1",
     "lodash": "^4.5.1",
-    "lzma-native": "^1.5.2",
+    "lzma-native": "^2.0.2",
```

Comment:
> also allows us to use a newer version of `node-gyp`

We don't need to depend on `node-gyp` at all – it ships with npm. Maybe you mean somthing else (bindings, nan)?

> Perhaps this change is worth doing anyway in a separate PR against the master branch?

Yeah, I'm just sorting out all the things we'll need to get this npm@5 thing greeeeeeeen.